src/libs/ReportUtils.ts

```diff
@@ -6323,6 +6323,10 @@ function canReportBeMentionedWithinPolicy(report: OnyxEntry<Report>, policyID: s
     return isChatRoom(report) && !isThread(report);
 }
 
+function shouldShowMerchantColumn(transactions: Transaction[]) {
+    return transactions.some((transaction) => isExpenseReport(allReports?.[transaction.reportID] ?? {}));
+}
+
```

Comment:
This might be better suited for the SearchUtils file

Reply:
and its currently unused so I will ask @WojtekBoman if when we want to start using it and is it needed?